app.py

```diff
@@ -436,12 +445,18 @@ def project_profile(project_id, msg):
     for row in rows_got_from_group_worksheet:
         management_groups.append([x.value for x in row])
 
+    # today = date.today()
+    # today = today.strftime("%m/%d/%y")
+    # now = datetime.now()
+    # current_time = now.strftime("%H:%M:%S")
+    #n = datetime.datetime.now().strftime("%Y-%m-%d_%H-%M-%S")
+
+
     records = EmailSendingRecord.query.filter_by(project_name=project.project, project_owner=current_user.username).all()
     if records is not None:
         sending_info_dict = {}
         for record in records:
-            sending_info_dict[record.eva_name] = [record.num_of_tasks, record.num_of_finished_tasks]
-        print(sending_info_dict)
+            sending_info_dict[record.eva_name] = [record.num_of_tasks, record.num_of_finished_tasks, record.time_email_sent]
```

Comment:
time_email_sent is showing up on website missing some 0's

For example, this was send at 11:08 but it says 11:8

![image](https://user-images.githubusercontent.com/1979690/98394499-7b9f1e80-2020-11eb-8293-91b051bb77aa.png)